src/Nethermind/Nethermind.Evm/Witness/VerkleExecWitness.cs

```diff
@@ -88,26 +78,22 @@ public bool AccessForTransaction(Address originAddress, Address? destinationAddr
     /// <returns></returns>
     public bool AccessForGasBeneficiary(Address gasBeneficiary, ref long gasAvailable)
     {
-        if (!AccessVersion(gasBeneficiary, ref gasAvailable)) return false;
-        if (!AccessNonce(gasBeneficiary, ref gasAvailable)) return false;
-        if (!AccessBalance(gasBeneficiary, ref gasAvailable)) return false;
+        if (!AccessBasicData(gasBeneficiary, ref gasAvailable)) return false;
         if (!AccessCodeHash(gasBeneficiary, ref gasAvailable)) return false;
-        if (!AccessCodeSize(gasBeneficiary, ref gasAvailable)) return false;
         // _logger.Info($"AccessCompleteAccount: {address.Bytes.ToHexString()} {isWrite} {gas}");
         return true;
     }
 
     public bool AccessForCodeOpCodes(Address caller, ref long gasAvailable)
     {
-        if (!AccessVersion(caller, ref gasAvailable)) return false;
-        if (!AccessCodeSize(caller, ref gasAvailable)) return false;
+        if (!AccessBasicData(caller, ref gasAvailable)) return false;
         // _logger.Info($"AccessForCodeOpCodes: {caller.Bytes.ToHexString()} {gas}");
         return true;
     }
 
     public bool AccessForBalance(Address address, ref long gasAvailable, bool isWrite = false)
```

Comment:
rename this to AccessForBalanceOpCode - access for balance does not make sense now - either we access basic data leaf or codeHash